docs-requirements.txt

```diff
@@ -2,14 +2,6 @@ sphinx==7.1.2
 sphinx-rtd-theme==2.0.0rc4
 sphinx-autodoc-typehints==1.25.2
 
-# Need to install the api/sdk in the venv for autodoc. Modifying sys.path
```

Comment:
Are these not needed anymore? I believe READTHEDOCs [config](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/.readthedocs.yml#L15) reads from this.

Reply:
You're right. Added a comment instead.